data/translations_en.json

```diff
@@ -0,0 +1,412 @@
+{
+  "en": {
+    "AcknowledgmentsItemsTitle": "Acknowledgment",
+    "AcknowledgmentsItemsDescription": "Acknowledges contributions by describing those that contributed.",
+    "AcknowledgmentsItemsNamesTitle": "List of acknowledged names",
+    "AcknowledgmentsItemsNamesDescription": "Contains the names of entities being recognized.",
+    "AcknowledgmentsItemsNamesItemsTitle": "Name of entity being recognized",
```

Comment:
Please update this according to the latest version of: https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/json_schema/csaf_json_schema.json

If you use a script to generate that (which I highly recommend), please also add the script to the PR.

Reply:
I updated the phrases according to the differences introduced since I created this file.

Yes, I used a script for initializing the file which I will add. However, there are some manual adjustments made, e.g. adding the CVSS labels.